tests/attacks/evasion/test_overload_attack.py

```diff
@@ -0,0 +1,76 @@
+# MIT License
+#
+# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2018
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import logging
+
+import numpy as np
+import pytest
+
+from art.attacks.evasion import OverloadPyTorch
+from tests.utils import ARTTestException
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.only_with_platform("pytorch")
+def test_generate(art_warning):
+    try:
+        import torch
+        model = torch.hub.load('ultralytics/yolov5:v7.0',  model='yolov5s')
+        x = np.random(0.0, 1.0, size=(100, 3, 640, 640))
+
+        attack = OverloadPyTorch(model,
+                                 eps = 16.0 /255.0,
+                                 max_iter = 5,
+                                 num_grid = 10,
+                                 batch_size = 1)
+
+        x_adv = attack.generate(x)
+
+        assert x.shape == x_adv.shape
```

Comment:
Could you please add additional asserts to check the expected values in `x_adv`?